patroni/ha.py

```diff
@@ -687,18 +688,21 @@ def _is_healthiest_node(self, members, check_replication_lag=True):
                         logger.info('Ignoring the former leader being ahead of us')
         return True
 
-    def is_failover_possible(self, members, check_synchronous=True):
+    def is_failover_possible(self, members, only_syncronous=False, cluster_lsn=None):
```

Comment:
The default of the flag changed. This function is called from `manual_failover_process_no_leader` without any flag. So it does not matter in that case - and can be false (because no leader)?